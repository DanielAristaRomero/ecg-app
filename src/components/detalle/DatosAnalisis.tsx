export default function DatosAnalisis() {
  return <div>DatosAnalisis</div>;
}
