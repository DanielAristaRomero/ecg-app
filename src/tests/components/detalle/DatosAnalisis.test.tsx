import { shallow } from "enzyme";
import DatosAnalisis from "../../../components/detalle/DatosAnalisis";

describe("Pruebas sobre DatosAnalisis.tsx", () => {
  test("debe rendirizar el componente correctamente", () => {});
  test("debe rendirizar el componente correctamente", () => {});
  test("debe rendirizar el componente correctamente", () => {});
  test("debe rendirizar el componente correctamente", () => {});
  test("debe rendirizar el componente correctamente", () => {});
  test("debe rendirizar el componente correctamente", () => {});
  test("debe rendirizar el componente correctamente", () => {});
  test("debe rendirizar el componente correctamente", () => {});
  test("debe rendirizar el componente correctamente", () => {
    const wrapper = shallow(<DatosAnalisis />);
    expect(wrapper).toMatchSnapshot();
  });
});
