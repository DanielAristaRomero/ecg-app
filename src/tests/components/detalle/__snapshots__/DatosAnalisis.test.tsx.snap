// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre DatosAnalisis.tsx debe rendirizar el componente correctamente 1`] = `ShallowWrapper {}`;
